tools/interactive/interactivetool_blobtoolkit.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="interactive_tool_blobtoolkit" tool_type="interactive" name="Interactive BlobToolKit" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>genome assembly QC viewer</description>
+    <macros>
+        <token name="@TOOL_VERSION@">4.1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/galaxy/blobtoolkit-server:@TOOL_VERSION@</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="Blobtoolkit View" requires_domain="True">
+            <port>80</port>
+            <url>view/all</url>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code"><![CDATA[
+    ## The following is a hack. Our embedded pulsar is not respecting the Galaxy jwd-tmp dir.
+    export TEMP=\$PWD &&
+    export TMP=\$PWD &&
+    export TMPDIR=\$PWD &&
```

Comment:
Is this some old copy and paste ? If there is a problem with embeded pulsar and the temp dir handling issue should be opened.